platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/editor/StateEditorSave.java

```diff
@@ -57,7 +57,9 @@ public void run() {
                 Files.createDirectories(directory);
             }
 
-            writeSignedStateFilesToDirectory(directory, getStateEditor().getSignedStateCopy());
+            final long selfId = -1;
```

Comment:
-1?

Reply:
If you save a state using the state editor, you don't have a node ID. I am currently using -1 to indicate "no node ID". Would you like me to formalize this using some constant in the API?